### PR TITLE
Update service image references

### PR DESCRIPTION
**Pull request by BytesGuy**
PR to update references to old database images to the next gen images

### CODE REVIEW

**Review by felicianotech — APPROVED** [review]

**Comment by rosieyohannan** [comment]
@Mayuko-Sa could you look at this when you have time? Should we ask @BytesGuy to remove the code snippet change in the Japanese content so that can be handled through crowdin? Thank you! :-D

**Comment by rosieyohannan** [comment]
@BytesGuy Could you check on the conflict here when you have time? I'll ping the translations team too :-D

**Comment by Mayuko-Sa** [comment]
Updated Crowdin on ja changes.

**Comment by devinmarieb** [comment]
@BytesGuy and @rosieyohannan I've resolved the conflicts, but I noticed the example on the databases page is using `image: cimg/python:3.10-browsers` without implementing the actual orb or installing it. I don't think this is a valid example that we can merge without that addition.

**Comment by rosieyohannan** [comment]
Sorry only just seen this @devinmarieb ... looks to me like the `-browsers` tag isn't strictly necessary so maybe we can just remove that to keep it simple. @BytesGuy?

**Comment by BytesGuy** [comment]
Sorry I missed the alert on this too. Yes the browsers tag is unnecessary here, not even sure how it sneaked in there! Feel free to remove it 😄

File: jekyll/_cci2/databases.md [hunk 1]
```diff
@@ -43,30 +43,31 @@ Set the POSTGRES_USER environment variable in your CircleCI config to `postgres`
...
 version: 2.1
+
 jobs:
   build:
 
     # Primary container image where all commands run
-
     docker:
-      - image: cimg/python:3.10.0
+      - image: cimg/python:3.10-browsers
```
Comment (rosieyohannan) [inline jekyll/_cci2/databases.md:52]:
```suggestion
      - image: cimg/python:3.10
```

**Pull request merged**